### ISSUE TITLE
[Bug] Teleporting Station modules?

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#2388]
### Short Description of the issue:

I'm having a hard time understanding this one myself. When I spawned in a bunch of my station modules to assemble, I put a few of them together, I realized I placed one wrong so I went to click on the correct connector, then move it to the spot I wanted it, when I looked at the new connector I wanted to attach to, myself and the module teleported. I hadn't clicked the second connector yet, I had only looked at it. 

### Steps to reproduce the issue:

Spawn in some station modules. 
Connect 3 of them together.
Using the connect tool, select a connector that isnt connected to anything, go and try and connect to a connector on a different module that is already connected to something.
Teleportation?

### Environment
- Starship EVO version: 20w24a
- Graphics Card: 2080 Super OC

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4762645/output_log.txt)

**Comment by tsunamayo** [comment]
Yes I also have noticed some random player teleportation, I still need a clear test case on how that happens. I dont think it is related to docking though, maybe more moving close to another entity?

**Comment by Sullos** [comment]
I have continued trying, and whilst it may not be related to docking specifically, I am able to recreate the bug repeatedly.  Here is another updated output log. Sometimes it's not just teleporting, I also went flying through space with the module at high speed. 
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4762786/output_log.txt)

**Comment by Sullos** [comment]
So I was simply building when suddenly I appeared on the other side of my station. I flew outside the station and realized I was in another part of the asteroid belt with none of my other stuff in sight. I then landed on my station module and I watched as we both flew into space. The only way to stop the flying away was to stop standing on the station module. I didn't have a speedometer visible but it was at a very high speed.
Another more updated output log:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4762930/output_log.txt)
Hopefully, this can be fixed. It's very odd and difficult to play with this teleporting/flying through space bug.
Let me know if I can help.

**Comment by Majikmonster** [comment]
I've been able to pinpoint (I think) exactly what's happening. Entering the gravity field of any module aligns the player to the field while the module simultaneously flies off in the direction the player was moving when they entered the field. The alignment on approach isn't enough to keep the player along for the ride when approaching too slow, as the character is not deep into the field enough and moves out of it on acceleration. I've been able to reliably reproduce this, and basically use station modules as weapons :P.

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4762940/output_log.txt)

![20200611015005_1](https://user-images.githubusercontent.com/51390907/84351810-171f8680-ab8a-11ea-8f6c-3f2f6023d3fb.jpg)
![20200611015409_1](https://user-images.githubusercontent.com/51390907/84351812-17b81d00-ab8a-11ea-98ad-891a35ff5a1e.jpg)
